resources/assets/js/components/CalendarAddModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <b-modal
+      id="calendar"
+      v-model="showModal"
+      no-stacking
+      no-body
+      size="lg"
+  >
+    <template slot="modal-title">
+      <slot name="title" />
+    </template>
+    <slot name="description" />
+    <b-input-group class="mt-2">
+      <b-input readonly size="lg" :value="copyUrl" ref="container" />
+      <b-input-group-append>
+        <b-button variant="white" class="butt" v-clipboard:copy="copyUrl" v-clipboard:success="copySucceed" v-clipboard:error="copyFail">
+          <b-img src="/images/copy.svg" />
+        </b-button>
+      </b-input-group-append>
+    </b-input-group>
+    <b-alert v-if="failed" show variant="danger" class="mt-2 mb-2">
+      <p class="m-0">
+        {{ translatedSomethingWrong }}
+      </p>
+    </b-alert>
+    <b-alert v-if="copied" show variant="info" class="mt-2 mb-2">
+      <p class="m-0">
+        {{ translatedCopied }}
+      </p>
+    </b-alert>
+    <div class="d-flex justify-content-between flex-wrap mt-4 mb-4">
+      <b-btn variant="link" href="https://talk.restarters.net/t/fixometer-how-to-add-repair-events-to-your-calendar-application/1770">
+        {{ translatedFindOutMore}}
+      </b-btn>
+      <b-btn variant="link" :href="editUrl">
+        {{ translatedSeeAll }}
+      </b-btn>
+    </div>
+    <template slot="modal-footer" slot-scope="{ cancel }">
```

Comment:
It's a way of passing things from the component into the slot so that they can then be accessed from within the slot.  The bootstrap modal component comes with its own cancel() method which closes the modal; here we pass it in to the slot and then invoke it on click.